packages/console/src/components/Table/index.tsx

```diff
@@ -9,19 +10,27 @@ import TableLoading from './TableLoading';
 import * as styles from './index.module.scss';
 import type { Column, RowGroup } from './types';
 
+export type TablePlaceholder = {
+  title?: string;
+  description?: string;
+  image?: ReactNode;
+  content?: ReactNode;
+};
+
 type Props<
   TFieldValues extends FieldValues = FieldValues,
   TName extends FieldPath<TFieldValues> = FieldPath<TFieldValues>
 > = {
   rowGroups: Array<RowGroup<TFieldValues>>;
   columns: Array<Column<TFieldValues>>;
   rowIndexKey: TName;
-  onClickRow?: (row: TFieldValues) => void;
+  isRowClickable?: (row: TFieldValues) => boolean;
+  rowClickHandler?: (row: TFieldValues) => void | undefined;
```

Comment:
`| undefined` is not necessary since the prop is already optional

Reply:
updated.